src/components/collapsible.tsx

```diff
@@ -30,6 +30,10 @@ interface Props {
   forceCollapse?: boolean
   /** Makes sure that the content is always in the DOM */
   alwaysRender?: boolean
+  /** Teal color chevron */
+  tealChevron?: boolean
+  /** Chevron position on the top for mobile and center for desktop */
+  mobileChevron?: boolean
```

Comment:
this sounds like micro-optimization for me and I would try getting around adding boolean props like this. I'd rather opt for a consistent behavior of the collapsible component instead of accommodating for subtle differences in where they are used